bindings/python/.evergreen/test.sh

```diff
@@ -82,6 +82,7 @@ for PYTHON_BINARY in "${PYTHONS[@]}"; do
     createvirtualenv $PYTHON_BINARY .venv
     python -m pip install --prefer-binary -r test-requirements.txt
     python -m pip install -v -e .
+    python synchro.py
```

Comment:
Could you open a new ticket to add a PR check to ensure we ran synchro.py before committing? It could even be part of pre-commit?

As part of that ticket we should be sure to remove this line otherwise we could be testing uncommitted code (eg when we forgot to run synchro.py before committing).